src/ts/ConvivaAnalyticsTracker.ts

```diff
@@ -607,6 +604,14 @@ export class ConvivaAnalyticsTracker {
     if (!this.isSessionActive() && !this.sessionEndedExternally) {
       this.internalInitializeSession();
     }
+
+    if (!this.hasPlayed) {
+      this.hasPlayed = true;
+      // Send the session init audio language values.
+      this.trackUpdateAudioTrack(this.player.getAudio());
+      // Check if at session init has a subtitle enabled.
+      this.trackInitialSubtitles();
+    }
```

Comment:
The current change preserves the initial implementation - https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/blob/v5.0.0/src/ts/ConvivaAnalytics.ts#L639, it's invoked inside `Play` event.

I guess the ready event does not cover the case if the play was never clicked in this case it would be called before the session is initialized (if I understand the sequence of events correctly READY -> PLAY -> PLAYING).

Also, I added:

<img width="687" alt="image" src="https://github.com/user-attachments/assets/110b0e1b-3040-414c-a7ca-1fc8cd3c9c0a">

but I don't see any output from the `Ready` event. 🤔

Reply:
> I guess the Ready event does not cover the case if the play was never clicked

Good call 🤔 Let's keep as is then 👍